phase1/master.cfg

```diff
@@ -881,7 +881,7 @@ for target in targets:
 		name = "dlprune",
 		description = "Pruning dl/",
 		descriptionDone = "dl/ pruned",
-		command = 'find dl/ -atime +15 -delete -print',
+		command = 'find dl/ -mindepth 1 -type f -atime +15 -delete -print',
```

Comment:
Good catch, however I did not use `-type f` because there are folders in there, e.g. in my tests `b43legacy-firmware-3.130.20.0` or `prism54-firmware-1`.
So I think `-mindepth 1` alone is the correct fix.
HTH

Reply:
> because there are folders in there
@f00b4r0 so we probably need to use `-exec rm -fr {}` as the `-delete` on directories wont work: 

```
 watching logfiles {}
 argv: b'find dl/ -mindepth 1 -atime +15 -delete -print'
 using PTY: False
find: cannot delete ‘dl/ath10k-ct-firmware-2020-11-08’: Directory not empty
program finished with exit code 1
```

Reply:
Hmm; I need to think about this more.
can you mark the step as `flunkOnFailure=False` in the meantime? this way these (harmless) failures will not register as build failures.

Reply:
Sure, done.